OptimizelySDK/Optimizely.cs

```diff
@@ -440,7 +443,26 @@ public Variation GetForcedVariation(string experimentKey, string userId)
             return DecisionService.GetForcedVariation(experimentKey, userId, config).ResultObject;
         }
 
-        #region  FeatureFlag APIs
+        public Variation GetFlagVariationByKey(string flagKey, string variationKey)
+        {
+            var flagVariationMap = ProjectConfigManager?.GetConfig().FlagVariationMap;
+            if (flagVariationMap.ContainsKey(flagKey) == true)
+            {
+                flagVariationMap.TryGetValue(flagKey, out var variations);
+
+                foreach (var variation in variations)
+                {
+                    if (variation.Key.Equals(variationKey))
```

Comment:
what about if Key is `null`, can't we directy check variation.Key == variationKey.

Reply:
Or Lambda / linq is ideal here.